mm2src/lp_ordermatch.rs

```diff
@@ -1522,10 +1523,13 @@ pub async fn orderbook(ctx: MmArc, req: Json) -> Result<Response<Vec<u8>>, Strin
     let base_coin: MmCoinEnum = try_s!(base_coin.ok_or("Base coin is not found or inactive"));
     let ordermatch_ctx: Arc<OrdermatchContext> = try_s!(OrdermatchContext::from_ctx(&ctx));
     let orderbook = try_s!(ordermatch_ctx.orderbook.lock());
+    let my_maker_orders = try_s!(ordermatch_ctx.my_maker_orders.lock());
+    let my_taker_orders = try_s!(ordermatch_ctx.my_taker_orders.lock());
     let asks = match orderbook.get(&(req.base.clone(), req.rel.clone())) {
         Some(asks) => {
             let mut orderbook_entries = vec![];
             for (uuid, ask) in asks.iter() {
+                let is_mine = my_maker_orders.get(uuid).is_some() || my_taker_orders.get(uuid).is_some();
```

Comment:
It might be not sufficient to check the local node `my_maker_orders` because it is not prohibited to run several nodes with same passphrase (not recommended though, but we may implement the proper multi-device support in the future). Please change this to `ask.pubsecp == our_pubkey` comparison. You can find the pubkey hex-encoding currently used at `PricePingRequest::new`. `my_maker_orders` and `my_taker_orders` data access will be simply not required after this change is made.

Reply:
Ok, thanks! I thought to compare addresses, but this way fits better for the multi-device support